cmd/gpu_nfdhook/labeler.go

```diff
@@ -87,6 +87,12 @@ func (l *labeler) scan() ([]string, error) {
 			continue
 		}
 
+		dat, err = ioutil.ReadFile(path.Join(l.sysfsDRMDir, f.Name(), "device/sriov_numvfs"))
```

Comment:
just a thought but would it make sense to have a helper utility somewhere in `pkg/internal` that could be used by the plugins (the gpu plugin has the same check IIRC) to check if VFs are enabled.

Reply:
Commit message could say that this NFD labeler functionality corresponds to earlier GPU plugin change.

Potential name of the common helper function could be isSriovPFwithVFs().

(At some later point, the relevant copyright lines in the files could be updated to list also 2021.)

Reply:
@mythi Not possible. "Internal packages (packages that are inside a folder that has an internal folder in their path) can only be imported from packages rooted at the parent of the internal folder."
I could perhaps introduce yet another "internal" folder under cmd, but given the makefiles, that might not work either.

Reply:
Makefile needs this: "cmds = $(shell ls --ignore=internal cmd)"